ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx

```diff
@@ -88,14 +89,22 @@ class ContentPanel extends Component {
 
   render() {
     const { stores, changeFilter, showDeleteConfirmation, showEditName, addNewWizard } = this.props;
-    const { devicesStore, featuresStore, groupsStore } = stores;
+    const { featuresStore, devicesStore, groupsStore } = stores;
```

Comment:
Why changing the order?

Reply:
Oops, forgot to revert this change, done :+1: